DP02_13a_Image_Cutout_SciDemo.ipynb

```diff
@@ -0,0 +1,1286 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.    diaSrc.sort('midPointTai')
</pre><p>You could do this in the query itself by including <code>ORDER BY midPointTai </code>in the query.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/171/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>

Reply:
<p>Done.</p>